params/version_test.go

```diff
@@ -0,0 +1,19 @@
+package params_test
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"testing"
+)
+
+func TestVersionFormat(t *testing.T) {
+	data, err := ioutil.ReadFile(filepath.Join("..", "VERSION"))
+	if err != nil {
+		t.Error("unable to open VERSION file")
+	}
+	matched, _ := regexp.Match(`^\d+\.\d+\.\d+(-[.\w]+)?\n?$`, data)
```

Comment:
Is the new line required? `^\d+\.\d+\.\d+(-[.\w]+)?$` should apply per line, right?

Reply:
It did not work without the newline.